snapper.spec.in

```diff
@@ -107,6 +107,8 @@ make %{?_smp_mflags}
 %install
 %make_install
 rm -f "%{buildroot}/%{_libdir}"/*.la "%{buildroot}/%{_lib}/security/pam_snapper.la"
+rm -f %{buildroot}/etc/cron.hourly/suse.de-snapper
+rm -f %{buildroot}/etc/cron.daily/suse.de-snapper
```

Comment:
Better remove the install in Makefile.am.

Reply:
Exactly, this only affects our distro...